source/speechViewer.py

```diff
@@ -109,4 +108,7 @@ def deactivate():
 	global _guiFrame, isActive
 	if not isActive:
 		return
+	# #7077: If the window is destroyed, text control wil be gone, so save speech viewer position before destroying the window.
+	_guiFrame.savePositionInformation()

```

Comment:
I think this was in the `onDestroy` function so that when windows initiates the closing of the dialog then the position information is still saved.